docs/Config_Reference.md

```diff
@@ -4215,6 +4216,32 @@ spi_software_miso_pin:
 ...
 ```
 
+#### aip31068_spi display
+
+Information on configuring an aip31068_spi display - a very similar to
+hd44780_spi display but uses 9 bit SPI commands.
+
+To avoid modifying SW_SPI driver instead sending 9-bit command, 8 commands packed in series and it sums to full 9 bytes without need for exoctic drivers.
```

Comment:
FWIW, I don't think the config reference is a good place to describe the implementation details.  Users configuring the display wont care how it is implemented.

Reply:
removed